version.php

```diff
@@ -30,6 +30,7 @@
 $plugin->version   = 2020101301;        // The current plugin version (Date: YYYYMMDDXX)
 $plugin->release   = 2020101301;        // The current plugin version (Date: YYYYMMDDXX)
 $plugin->requires  = 2016021800;        // Requires this Moodle version.
+$plugin->supported = [310, 400];
```

Comment:
When you mention 310+ in the README, shouldn't this also test `master` (401) for completness?